examples/with-google-analytics/pages/_app.js

```diff
@@ -14,7 +15,28 @@ const App = ({ Component, pageProps }) => {
     }
   }, [router.events])
 
-  return <Component {...pageProps} />
+  return (
+    <>
+      {/* Global Site Tag (gtag.js) - Google Analytics */}
+      <Script
```

Comment:
Would be nice to add the loading strategy to make it in sync with the documentation(even if afterInteractive is default)

Reply:
Makes sense